hbase-thrift/src/test/java/org/apache/hadoop/hbase/thrift/TestThriftServer.java

```diff
@@ -871,4 +878,32 @@ private void closeScanner(
     handler.scannerGet(scannerId);
     handler.scannerClose(scannerId);
   }
+
+  @Test
+  public void testGetThriftServerType() throws Exception {
```

Comment:
I probably missed this in the last iteration, what is the value add of doing both this and below? 

UTIL starts a full fledged cluster (which includes a proper thrift server), why run another thrift server in a thread and running the getThriftServerType() test? My point is that the patch can be simplified.

Reply:
Util doesn't seem to start the thriftserver. We need to test that thrift server can return the correct server type, so we started a thriftserver as in other unit tests of thrift.

Reply:
Ah you are right, HBaseHandler is a wrapper around normal client. Thanks.